macaroon.js

```diff
@@ -992,7 +992,7 @@ const importJSONV1 = function(obj) {
  * @returns {Macaroon}
 */
 const importJSONV2 = function(obj) {
-  if (obj.v !== 2) {
+  if (obj.v && obj.v !== 2) {
```

Comment:
Please add a comment:

    // The Go macaroon library omits the version, so we'll assume that
    // it is 2 in that case. See https://github.com/go-macaroon/macaroon/issues/35

Also, I'm not sure tihs will work as intended if v is `0` or `""`. How about:

    if (obj.v !== 2 && obj.v !== undefined) {

@hatch thoughts?

Reply:
Sorry, yes that is an improvement, I had assumed that 0 was an invalid value. This is more explicit 👍

Reply:
0 is an invalid value, but the originally proposed logic would have treated it the same as if it was 2 AFAICS.